libs/dh/esett/feature-balance-responsible/src/lib/drawer/dh-drawer.component.html

```diff
@@ -17,32 +17,44 @@
 <ng-container *transloco="let t; read: 'eSett.balanceResponsible.shared'">
   <watt-drawer #drawer (closed)="onClose()">
     <watt-drawer-heading>
-      <h2 class="message-heading">{{ balanceResponsibleMessage?.supplier?.value }}</h2>
+      <h2 class="message-heading">{{ balanceResponsibleMessage?.supplierWithName?.value }}</h2>
 
-      <watt-description-list [groupsPerRow]="3" variant="flow">
+      <watt-description-list *ngIf="balanceResponsibleMessage" [groupsPerRow]="4" variant="flow">
         <watt-description-list-item
-          [label]="t('validFrom')"
-          [value]="balanceResponsibleMessage?.validFromDate | wattDate"
+          [label]="t('electricitySupplier')"
+          [value]="balanceResponsibleMessage.supplierWithName.value"
         />
         <watt-description-list-item
-          [label]="t('validTo')"
-          [value]="balanceResponsibleMessage?.validToDate | wattDate"
+          [label]="t('balanceResponsible')"
+          [value]="balanceResponsibleMessage.balanceResponsibleWithName.value"
         />
         <watt-description-list-item
-          [label]="t('received')"
-          [value]="balanceResponsibleMessage?.receivedDateTime | wattDate"
+          [label]="t('gridArea')"
+          [value]="balanceResponsibleMessage.gridArea.code"
```

Comment:
Question: Not Name - Code?

Reply:
😱 Good catch 

It is actually supposed to be "name - code". Adding it in a moment.